sos/report/plugins/snapper.py

```diff
@@ -0,0 +1,24 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from glob import glob
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Snapper(Plugin, IndependentPlugin):
+
+    short_desc = 'System snapper'
+
+    plugin_name = 'snapper'
+    profiles = ('system', 'hardware')
+
```

Comment:
There needs to be some enablement triggers here. `packages`, or `files`, or `commands`, etc... Without one, this will always be enabled.

Reply:
ok.. will update this
    short_desc = 'System snapper'

    plugin_name = 'snapper'
    commands = ("snapper",)